pengwin-setup.d/programming.sh

```diff
@@ -15,6 +15,7 @@ function main() {
       "DOTNET" "Install .NET Core SDK from Microsoft and optionally install NuGet  " off \
       "GO" "Install the latest Go from Google" off \
       "JAVA" "Install the SDKMan to manage Java SDKs" off \
+      "JETBRAINS" "Install required support to jetbrains tools" off \
```

Comment:
@javiersantossanchez also you need to increment the list vertival size to avoid the scroll bar